plugin/package.json

```diff
@@ -56,5 +56,13 @@
   },
   "msw": {
     "workerDirectory": "web/mock"
+  },
+  "dependencies": {
+    "@ant-design/icons": "^4.7.0",
+    "antd": "^4.21.7",
```

Comment:
Like I mentioned yesterday antd should be a dev dependency. 
And if @ant-design/icons won't build properly please look into the CDN plugin I told you about for whether or not ant-design/icons are supported by CDN